src/components/Chart/Display.svelte

```diff
@@ -3,12 +3,14 @@
     import Chart from 'chart.js/auto';
     import ChartStreaming from 'chartjs-plugin-streaming';
     import ChartZoom from 'chartjs-plugin-zoom';
-    import { onMount } from 'svelte';
+    import { onDestroy, onMount } from 'svelte';
```

Comment:
This is apparently unused.
```suggestion
    import { onMount } from 'svelte';
```